language/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -137,7 +139,8 @@ pub trait MoveTestAdapter<'a>: Sized {
         args: Vec<<<Self as MoveTestAdapter<'a>>::ExtraValueArgs as ParsableValue>::ConcreteValue>,
         gas_budget: Option<u64>,
         extra: Self::ExtraRunArgs,
-    ) -> Result<(Option<String>, SerializedReturnValues)>;
+    ) -> Result<(Option<String>, SerializedReturnValues, Option<Value>)>;
```

Comment:
Can we remove the `Option<Value>` from everything other than view?

Reply:
every command needs the context value, not only the `view`, the `view` command just an example.